README.md

```diff
@@ -28,6 +28,12 @@ To install with npm:
 npm install chosen-js
 ```
 
+To install with composer:
+
+```
+composer require harvesthq/chosen
```

Comment:
For this to work it should still be added to packagist right?

Reply:
Sure, it will after merge.